pkg/operator/ceph/object/user/controller.go

```diff
@@ -89,6 +89,11 @@ func (c *ObjectStoreUserController) StartWatch(stopCh chan struct{}) error {
 }
 
 func (c *ObjectStoreUserController) onAdd(obj interface{}) {
+	if c.clusterSpec.External.Enable && !c.clusterSpec.External.RunStatelessWorkload {
+		logger.Warningf("Creating object store user for an external ceph cluster is disabled (RunStatelessWorkload is false)")
```

Comment:
Nit: from the user-facing side of things, this is `runStatelessWorkload` not capitalized.